src/ProophessorDo/Infrastructure/Repository/EventStoreTodoList.php

```diff
@@ -1,42 +1,30 @@
 <?php
-/*
- * This file is part of prooph/proophessor.
- * (c) 2014-2015 prooph software GmbH <contact@prooph.de>
+/**
+ * This file is part of prooph/proophessor-do.
```

Comment:
-do-symfony

Reply:
yep, it is just copied from prooph-do. This is only the next step to a merged repo. I'd say we keep the headers as-is, because we will use a shared model in the future anyway.